config/libvirt-setup/setup-system.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
I'm not a pro on ansible roles, but can't libvirt-setup just be another role under config/duffy-setup?  Side note, I believe the plan when we move to jenkins pipeline is to provision one duffy node for the entire pipeline, so we are going to have to combine our requirements for the duffy host instead of having separate files.